test/llvm-integration/LLVM.hs

```diff
@@ -90,6 +91,10 @@ llvmSpec =
                 it "should work with latin-1strings" $
                     hedgehog . propertyTest . latin1Prop
 
+            describe "Correct sort injections in non KItem maps" $
+                it "should work with latin-1strings" $
```

Comment:
Thinking about it again, ... We are running this 2000 times (`modifyMaxSuccesses (* 20)` above, default 100 AFAIK).
Maybe it would work to just repeat the `beforeAll loadAPI` here? And we can remove the `describe`
```suggestion
        beforeAll loadAPI $
            it "should correct sort injections in non KItem maps" $
```